azure-kusto-data/azure/kusto/data/helpers.py

```diff
@@ -50,9 +52,17 @@ def dataframe_from_result_table(table: "Union[KustoResultTable, KustoStreamingRe
     for col in table.columns:
         if col.column_type == "bool":
             frame[col.column_name] = frame[col.column_name].astype(bool)
-        if col.column_type == "datetime":
+        elif col.column_type == "int" or col.column_type == "long":
+            frame[col.column_name] = frame[col.column_name].astype("Int64")
+        elif col.column_type == "real" or col.column_type == "decimal":
```

Comment:
Should we also check for "double" since it's an alias (https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/scalar-data-types/)

(I guess we don't check for "date" and "time" either)

Reply:
The alias is for user comfort, the type underneath will be "real" and the column type in the response will be "real" even if the user defined it as "double".